src/components/toast/global_toast_list.tsx

```diff
@@ -48,14 +52,28 @@ export interface EuiGlobalToastListProps extends CommonProps {
    * Determines which side of the browser window the toasts should appear
    */
   side?: ToastSide;
+  /**
+   * At this threshold, a "Clear all" button will display at the bottom of the toast list
+   * that allows users to dismiss all toasts in a single click.
+   *
+   * Defaults to `3`. Set to `0` to disable the button entirely.
+   */
+  showClearAllButtonAt?: number;
+  /**
+   * Optional callback that fires when a user clicks the "Clear all" button.
+   * Ignored if `showClearAll` is not `true`.
```

Comment:
We can remove this documentation line that we're adding the button by default (🎉), and due to the changed corresponding prop
```suggestion
   * Ignored if `showClearAll` is not `true`.
```